.github/workflows/pull-request.yml

```
@@ -295,7 +295,48 @@ jobs:
         with:
           path: './coverage/lcov.info'
           min_coverage: 95.0
-          
+  
+  Docker-Check:
+    needs: Test-Application
+    runs-on: ubuntu-latest
+    steps: 
```

yamllint

[error] trailing spaces

(trailing-spaces)